scanner/src/file_results.rs

```diff
@@ -85,7 +85,7 @@ pub struct FileResults {
 
     /// The uncompressed length of the SWF file.
     #[serde(rename = "Uncompressed Length")]
-    pub uncompressed_len: Option<u32>,
+    pub uncompressed_len: Option<i32>,
```

Comment:
This doesn't really make much sense here (or possibly in some other places) - would it be good to leave it as a `u32` and do relevant coercions in places where they're needed?

Reply:
`uncompressed_len` needs to be an `i32` because in the error state `getBytesTotal()` returns `-1`. And `getBytesTotal()` returns `uncompressed_len`. Because there also exist valid states in which `getBytesTotal()` returns `0`, we can't just return `-1` if uncompressed_len is `0`.

To implement the error state correctly as Flash player does it, `uncompressed_len? needs to be `i32` so it can be set to `-1` in the error case.